ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -126,6 +191,52 @@ export const CoinOverview = ({
   }, [isMarketingEnabled, isMetaMetricsEnabled, metaMetricsId, trackEvent]);
   ///: END:ONLY_INCLUDE_IF
 
+  const renderPercentageAndAmountChange = () => {
+    if (isEvm) {
```

Comment:
Do you know how we can test the non-evm / bitcoin case?  Or have screenshots showing it doesn't break before / after this change?

Reply:
I made sure i did not break their tests; Wanted to get assets feedbacks first then check with their team to QA this feature 🙏